### ISSUE TITLE
DL3062 False positive on `go run` with versions pinned

### ISSUE THREAD

**Issue by LiorHalfon** [hadolint/hadolint#1134]
I got a false positive on this line:

```RUN --mount=type=secret,id=gcloud,target=/key.json GOPROXY=proxy.golang.org go run github.com/GoogleCloudPlatform/artifact-registry-go-tools/cmd/auth@v0.1.0 add-locations --locations=europe-west4 && go run github.com/GoogleCloudPlatform/artifact-registry-go-tools/cmd/auth@v0.1.0 refresh```

Got this error:
```
Pin versions in go. Instead of `go install <package>` use `go install <package>@<version>`
```
But as you can see, both commands have "@".
@Danil42Russia

**Comment by m-ildefons** [comment]
Hi,
thanks for the bug report. I believe this will be fixed by this PR: https://github.com/hadolint/hadolint/pull/1131
